sdk/src/test/java/com/horizen/bouncycastle/BouncyCastleTest.java

```diff
@@ -0,0 +1,132 @@
+package com.horizen.bouncycastle;
+
+import com.horizen.utils.Pair;
+import org.bouncycastle.math.ec.rfc8032.Ed25519;
+import org.bouncycastle.util.encoders.Hex;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.*;
+
+public class BouncyCastleTest {
+
+    @Test
+    public void testKeiPair() {
+        byte[] seed = "seed1".getBytes();
+        Pair<byte[], byte[]> kp1 = com.horizen.utils.Ed25519.createKeyPair(seed);
+        Pair<byte[], byte[]> kp2 = com.horizen.utils.Ed25519.createKeyPair(seed);
+        byte[] pvk = kp1.getKey();
+        byte[] pvk2 = kp2.getKey();
+        assertTrue(Arrays.equals(pvk, pvk2));
```

Comment:
please, add message descriptions to asserts. The same below

Reply:
Done